docs/content/preview/api/ysql/exprs/func_yb_hash_code.md

```diff
@@ -73,20 +74,22 @@ You can consider a modified version of the above `SELECT` query as follows:
 SELECT * FROM sample_table WHERE yb_hash_code(x,z) <= 128 AND yb_hash_code(x,z) >= 0;
 ```
 
-Here, the `yb_hash_code` calls are on `x` and `z`. `x` and `z` do not form the full hash key of the base sample_table table but they do form the full hash key of the index table, `sample_idx`. Hence, the optimizer will consider pushing down the calls to the DocDB layer if an index scan using `sample_idx` is chosen. Note that the optimizer may choose not to go with a secondary scan if it deems the requested hash range to be large enough to warrant doing a simple full table scan instead. The `EXPLAIN ANALYZE` result for this could be as follows
+Here, the `yb_hash_code` calls are on `x` and `z`. `x` and `z` do not form the full hash key of the base sample_table table but they do form the full hash key of the index table, `sample_idx`. Hence, the optimizer will consider pushing down the calls to the DocDB layer if an index scan using `sample_idx` is chosen. Note that the optimizer may choose not to go with a secondary scan if it deems the requested hash range to be large enough to warrant doing a full table scan instead. The `EXPLAIN ANALYZE` result for this could be as follows
 
 ```sql
 EXPLAIN ANALYZE SELECT * FROM sample_table WHERE yb_hash_code(x,z) <= 128 AND yb_hash_code(x,z) >= 0;
 ```
 
 ```output
-                                                         QUERY PLAN
-----------------------------------------------------------------------------------------------------------------------------
- Index Scan using sample_idx on sample_table  (cost=0.00..5.96 rows=16 width=12) (actual time=8.923..8.975 rows=18 loops=1)
-   Index Cond: (yb_hash_code(x, z) <= 128)
- Planning Time: 0.066 ms
- Execution Time: 9.033 ms
-(4 rows)
+                                                           QUERY PLAN
+---------------------------------------------------------------------------------------------------------------------------------
+ Seq Scan on sample_table  (cost=10000000000.00..10000000020.00 rows=1000 width=12) (actual time=10.259..63.165 rows=18 loops=1)
+   Filter: ((yb_hash_code(x, z) <= 128) AND (yb_hash_code(x, z) >= 0))
+   Rows Removed by Filter: 9982
+ Planning Time: 5.970 ms
+ Execution Time: 63.574 ms
```

Comment:
Oops! Looks like I had the wrong output here. Mixed from the other peak memory PR. This one must have an index scan anyway because we are creating indexes here. Will include the correct outputs. Thank you!